curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/65004ba581d03d1d5628b41c.md

```diff
@@ -298,6 +298,7 @@ const taskData = JSON.parse(localStorage.getItem("data")) || [];
 let currentTask = {};
 
 const addOrUpdateTask = () => {
+  addOrUpdateTaskBtn.innerText = "Add Task";
```

Comment:
It looks like you added the solution only to step 47 and 48.
But you need to add the solution for all remaining steps to the end. Ex. 49, 50,51,52, etc
You also need to add the solution to step 58's final solution code in the `--solutions` like mentioned in the issue.